docs/data/toolpad/building-ui/canvas-and-inspector.md

```diff
@@ -0,0 +1,43 @@
+# Canvas
+
+<p class="description">
+    Canvas and Inspector are the two main areas when it comes to UI building
+</p>
+
+## Working with canvas
+
+Building UI is as easy as dragging component from the library and dropping it on the canvas.
+
+Once you have a few components on the canvas you can easily tweak them by doing following actions:
+
+<img src="/static/toolpad/canvas-controls.png" width="600px" alt="Canvas controls" />
+
+- Select component - **click** on any component
+- Remove component from canvas - **click** bin icon in the component's hud
+- Changing component's order/placement on the canvas - initiate draggin by **clicking** on the component's hud when you see mouse cursor change to a hand icon
+
+## Inspector
+
+Upon component selection you can configure a set of predefined props (each component may have a different set of configurable props) in the inspector on the right side
+
+<img src="/static/toolpad/inspector.png" width="270px" alt="Inspector" />
+
+You can either use a static values or use dynamic ones as described in [data binding](/toolpad/data-binding/) section
+
+## Canvas grid
+
+In order to make UI building more flexible you can use not only the row layout but add components side by side and use a different columns.
```

Comment:
mentioning stacking elements vertically should maybe be separate from the mentions about resizing?

Reply:
Do you have any specific suggestion how to formulate that? I put it like that because you can't really resize elements unless you first stack them vertically? 🤔

Reply:
Maybe something like? (just an idea):

## Resizing elements horizontally
You can grab components on the side edges and resize them within the grid boundaries.

## Column layouts
Besides using a row layout where elements are placed side by side, you can stack elements vertically in the same column.

Reply:
Check it out now, I tweaked a little bit and also added more screenshots

Reply:
looks better now!